frontend/src/components/visual-editor/form/ListMessageForm.tsx

```diff
@@ -186,7 +186,9 @@ const ListMessageForm = () => {
               return (
                 <AutoCompleteSelect<ContentField, "label", false>
                   options={(contentType?.fields || []).filter(
-                    ({ type }) => ContentFieldType.TEXT === type,
+                    ({ type }) =>
+                      ContentFieldType.TEXT === type ||
+                      ContentFieldType.TEXTAREA === type,
```

Comment:
Textarea should be available only in subtitle
```suggestion
                    ({ type }) => ContentFieldType.TEXT === type,
```